subiquity/server/controllers/install.py

```diff
@@ -312,6 +315,17 @@ async def run_curtin_step(name, stages, step_config, source=None):
                 step_config=self.generic_config(),
                 source=source,
                 )
+            if self.app.opts.dry_run:
+                # In dry-run, extract does not do anything. Let's create what's
+                # needed manually. Ideally, we would not hardcode
+                # var/lib/dpkg/status because it is an implementation detail.
+                status = Path("var/lib/dpkg/status")
+                (Path(root) / status).parent.mkdir(parents=True, exist_ok=True)
```

Comment:
This is nitpicky to the max but a) root is already a Path here b) Path / str works fine? so you can leave status as a string.

Reply:
Good point, updated :)